FV3/atmos_cubed_sphere/model/fv_mapz.F90

```diff
@@ -438,13 +481,31 @@ subroutine Lagrangian_to_Eulerian(last_step, consv, ps, pe, delp, pkz, pk,   &
 
    if ( .not. hydrostatic ) then
 ! Remap vertical wind:
+      !$ser verbatim if(j == js) then 
+      !$ser savepoint Map1_PPM_2d-In
+      !$ser verbatim mode=-2
+      !$ser data j_2d=js pe1=pe1 pe2=pe2 var_in=w ws_1d=ws(:,j) var_inout=w mode=mode
+      !$ser verbatim endif
         call map1_ppm (km,   pe1,  w,  ws(is,j),   &
                        km,   pe2,  w,              &
                        is, ie, j, isd, ied, jsd, jed, -2, kord_wz)
+      !$ser verbatim if(j == js) then 
+      !$ser savepoint Map1_PPM_2d-Out
+      !$ser data var_inout=w
+      !$ser verbatim endif
 ! Remap delz for hybrid sigma-p coordinate
+      !$ser verbatim if(j == js) then 
+      !$ser savepoint Map1_PPM_2d-In
```

Comment:
Is it a problem that this savepoint has the same name as the one above?

Reply:
Very good question! We modified Serialbox such that every savepoint gets a unique id in the metadata associated with it, which allows us to duplicate the savepoint name. This is handy for being able to test a particular method on several different use cases with different variables and arguments used.